app/workers/hard_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class HardWorker
```

Comment:
How about we use a more descriptive name for the worker? so it's easier to know what is function is without looking it up